Makefile

```diff
@@ -237,6 +241,43 @@ $(PYTHON_BIN)/activate: $(MAGE) script/requirements.txt
 $(APPROVALS):
 	@$(GO) build -o $@ github.com/elastic/apm-server/approvaltest/cmd/check-approvals
 
+##############################################################################
+# Rules for the package-storage.
+##############################################################################
```

Comment:
Can we make these scripts in `.ci/scripts`? This is entirely CI-related, not something that developers will run directly.

Reply:
I've just committed https://github.com/elastic/apm-server/pull/7900/commits/08fcd2f5146e68584e121179f51d61f3bdf27951 and tested it locally.

I'm gonna give a go in the CI and see how it goes